src/terrain/clients/iplant_groups.clj

```diff
@@ -49,21 +49,14 @@
     user-info
     (empty-user-info short-username)))
 
-(defn search-subjects
-  "Uses iplant-groups's subject search endpoint to retrieve user details."
-  [user search]
-  (let [res (http/get (str (curl/url (config/ipg-base) "subjects"))
-                      ;; Adding wildcards matches previous (trellis) search behavior
-                      {:query-params {:user user :search (str "*" search "*")}
```

Comment:
This is odd -- I recall that when I was doing this before, some stuff changed and some stuff didn't. It may be that the search methods in the grouper config have since changed, though, since I recall we're also able to control whether wildcards are added there or not.